redirects/management/commands/check_redirects.py

```diff
@@ -16,13 +16,15 @@ def handle(self, *args, **options):
         redirects = Redirect.objects.all()
         bad_redirects = ''
         dev_email_info = ''
+        USER_AGENT = 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.66 Safari/537.36'
+        BAD_HTML_CODES = [404, 410, 500, 503]
 
         #Loop through and validate
         for re in redirects:
             try:
-                response = requests.get(re.redirect_link, timeout=(10, 10), verify=certifi.where())
+                response = requests.get(re.redirect_link, timeout=(10, 10), verify=certifi.where(), headers={ "user-agent": USER_AGENT})
                 # if bad one is found, add short URL and redirect to list
-                if response.status_code != 200:
+                if response.status_code in BAD_HTML_CODES:
```

Comment:
this could be `if response.status_code not in range(400, 500):` depending on if you don't mind excluding all 4xx and 5xx errors

Reply:
403 is being returned on some redirects that actually work, so I wanted to be specific on the codes.